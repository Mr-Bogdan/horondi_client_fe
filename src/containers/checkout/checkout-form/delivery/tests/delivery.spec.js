import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Delivery from '../delivery';
import Courier from '../courier/courier';
import NovaPost from '../nova-post';

Enzyme.configure({ adapter: new Adapter() });

let wrapper;
beforeEach(() => {
  wrapper = shallow(<Delivery deliveryType='NOVAPOSTCOURIER' isLightTheme />);
});

describe('Delivery component tests', () => {
  it('should render one <Courier>', () => {
    expect(wrapper.find(Courier)).toHaveLength(1);
  });
  it('should render zero <NovaPost>', () => {
    expect(wrapper.find(NovaPost)).toHaveLength(0);
  });
  // it('should render zero <NovaPost>', () => {
  //     wrapper = shallow(<Delivery deliveryType='NOVAPOSTCOURIER' isLightTheme={false} />);
  //     expect(wrapper.find(Courier)).toHave;
  // });
  it('should handle change', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
