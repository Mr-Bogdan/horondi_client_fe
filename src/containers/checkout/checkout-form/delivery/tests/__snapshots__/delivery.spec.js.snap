// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Delivery component tests should handle change 1`] = `
<Fragment>
  <Courier
    deliveryType="NOVAPOSTCOURIER"
    isLightTheme={true}
  />
</Fragment>
`;
